build.gradle

```diff
@@ -65,6 +65,10 @@ dependencies {
     testImplementation "org.amshove.kluent:kluent:$kluent_version"
     testImplementation "org.spekframework.spek2:spek-dsl-jvm:$spek_version"
     testImplementation "io.ktor:ktor-server-test-host:$ktor_version"
+    testImplementation ("no.nav:kafka-embedded-env:1.0.0") {
```

Comment:
Update to 2.1.1

Reply:
Når jeg oppdaterer denne får jeg problemer med at den har en versjon av zookeeper som forventer at log4j finnes på classpath, og jeg har ikke så lyst til å dra med meg log4j inn i appen min... 😅 

no.nav.syfo.syfosmvarsel.avvistsykmelding.AvvistSykmeldingServiceKtTest STANDARD_ERROR
    Exception in thread "Thread-15" java.lang.NoClassDefFoundError: org/apache/log4j/jmx/HierarchyDynamicMBean
        at org.apache.zookeeper.jmx.ManagedUtil.registerLog4jMBeans(ManagedUtil.java:51)
        at org.apache.zookeeper.server.ZooKeeperServerMain.initializeAndRun(ZooKeeperServerMain.java:77)
        at no.nav.common.embeddedzookeeper.ZKServer$ZKS.start(ZKServer.kt:53)
        at no.nav.common.embeddedzookeeper.ZKServer$ZKS.access$start(ZKServer.kt:30)
        at no.nav.common.embeddedzookeeper.ZKServer$ZKS$zkThread$1.invoke(ZKServer.kt:58)
        at no.nav.common.embeddedzookeeper.ZKServer$ZKS$zkThread$1.invoke(ZKServer.kt:30)
        at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:30)